cluster/charts/crossplane/templates/clusterrole.yaml

```diff
@@ -77,3 +77,12 @@ rules:
   - patch
   - delete
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - create
+  - update
+  - watch
```

Comment:
@negz These verbs are needed for leader lease resource, technically we can add the rest of the verbs too, but I think omitting those should be fine.

Reply:
Sounds good! Will we need to add these to the RBAC manager's cluster role template too?

https://github.com/crossplane/crossplane/blob/master/pkg/controller/rbac/provider/roles/roles.go#L54

We'll probably need to add ConfigMaps to the above extra rules that the RBAC manager creates for all provider deployments, too.

Reply:
Yes that's a good catch, but it's kind of important when provider is also enabling leader election process. To make it simple for now I think it's fine to just enable it by default irrelevant of what provider wants, but maybe it will be good idea to extract this from provier itself (maybe new field `LeaderElection bool` in `PackageSpec`?)

Also to be in parity I'm gonna update `clusterrole.yaml` to include all the verbs of `verbsSystem` (i.e. `"get", "list", "watch", "update", "patch", "create"`).